saleor/graphql/shipping/mutations/channels.py

```diff
@@ -232,9 +233,7 @@ def clean_add_channels(cls, shipping_method, input):
 
     @classmethod
     def perform_mutation(cls, _root, info, id, input):
-        shipping_method = cls.get_node_or_error(
-            info, id, only_type=ShippingMethod, field="id"
-        )
+        shipping_method = get_shipping_model_by_object_id(id)
```

Comment:
This is mutation for managing the shipping_method. It is fine if this part of code will raise an error (it is even expected with inccorect ID). IMHO this part of code should work as previously - If shipping_method doesn't exist then we raise an error. If there is an issue with relation ShippingMethod type to ShippingMethod model, then I would overwrite the mutation method to handle such case. 
Currently we change the behaviour of Saleor, as your function can return None here, which were never happen before.